runchecks.sh

```diff
@@ -16,15 +20,10 @@ echo "Running isort"
 isort --check-only --diff --quiet \
  || { echo "isort errors found! Run 'isort' with no options to fix."; exit 1; }
 
-echo "Running shellcheck"
-git grep -El '^#!/.+\b(bash|sh)\b' | xargs shellcheck
+echo "Running shellcheck (if availablein PATH)"
+if hash shellcheck 2>/dev/null; then
+  git grep -El '^#!/.+\b(bash|sh)\b' | xargs shellcheck
+fi
 
 echo "Running test docs generation"
 mkdocs build
-
-echo "Running Django system checks"
-# See .travis.yml for explanation of the environment variable overriding.
-SITE_URL="https://treeherder.dev" TREEHERDER_DEBUG="False" python -bb ./manage.py check --deploy --fail-level WARNING
```

Comment:
I'm moving this out of here since I only want checks in this script that does *not* require spinning up any containers.